apps/editor/src/app/atoms/placeholderInput.tsx

```diff
@@ -23,7 +24,8 @@ export function PlaceholderInput({
   children,
   onAddClick,
   disabled,
-  maxHeight = 450
+  maxHeight = 450,
+  minHeight
```

Comment:
needed for some places where there's no specified height. So height: 100% did nothing.